src/popup/components/Tokens.vue

```diff
@@ -76,25 +85,29 @@ export default defineComponent({
   },
   setup(props) {
     function getAvailableCharLength() {
-      if (props.tokens?.length < 2) return props.symbolLength;
-      const shorterNameLength = [props.tokens[0].symbol.length, props.tokens[1].symbol.length]
-        .find((length) => length < props.doubleSymbolLength);
+      if (props.tokens?.length < 2) {
+        return props.symbolLength;
+      }
+      const shorterNameLength = [
+        props.tokens[0].symbol.length,
+        props.tokens[1].symbol.length,
+      ].find((length) => length < props.doubleSymbolLength);
```

Comment:
What are your thoughts of removing that additional assignment?
```suggestion
      const shorterNameLength = props.tokens.find(({ symbol }) => symbol.length < props.doubleSymbolLength);
```

Reply:
OK, makes sense. But I did it in a little different way - mapping the tokens and then finding.